scipy/stats/_sobol.pyx

```diff
@@ -91,11 +91,19 @@ def initialize_direction_numbers():
         np.savez_compressed("./_sobol_direction_numbers", vinit=vs, poly=poly)
 
     """
-    global is_initialized, poly, vinit
```

Comment:
`poly, vinit` do not need to be `global`? Just so I understand how this works 😅

Reply:
The code does `poly[:] = ...` rather than `poly = ...` so the variable lookup goes from local to global scope trying to find `poly` and finds it at the module level.

Reply:
Ah yes, got it! Thanks Matti

Reply:
Ah yes, got it! Thanks Matti